python/datascience/requirements-below-3.11.txt

```diff
@@ -16,4 +16,5 @@ sympy==1.11.*
 squarify==0.4.*
 tensorflow>=2.10,<3
 torch==1.12.*
-torchvision==0.13.*
\ No newline at end of file
+torchvision==0.13.*
+snowflake-snowpark-python
```

Comment:
Does this not need a version?

Reply:
Good point, this package is a bit 💩 and some time ago it only supported Python 3.8. But according to a build, all Python versions now installed are the latest 1.26.0, so I will mark that in requirements.txt to prevent magic updates.